prdoc/pr_6643.prdoc

```diff
@@ -0,0 +1,44 @@
+title: Added fallback_max_weight to Transact for sending messages to V4 chains
+doc:
+- audience: Runtime Dev
+  description: |-
+    Removing the `require_weight_at_most` parameter in V5 Transact had only one problem. Converting a message from V5 to V4 to send to chains that didn't upgrade yet. The conversion would not know what weight to give to the Transact, since V4 and below require it.
+
+    To fix this, I added back the weight in the form of an `Option<Weight>` called `fallback_max_weight`. This can be set to `None` if you don't intend to deal with a chain that hasn't upgraded yet. If you set it to `Some(_)`, the behaviour is the same. The plan is to totally remove this in V6 since there will be a good conversion path from V6 to V5.
```

Comment:
```suggestion
    To fix this, XCM v5 Transact now also takes a `fallback_max_weight: Option<Weight>` parameter. This can be set to `None` if the instruction is not meant to be sent to chains running XCM versions lower than V5. If set to `Some(w)`, a subsequent conversion to V4 will result in `Transact { require_weight_at_most: w, .. }`. The plan is to remove this workaround in V6 since there will be a good conversion path from V6 to V5.
```